.github/workflows/labeler.yml

```
@@ -0,0 +1,15 @@
+name: Labeler
+on: pull_request
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+    - uses: actions/labeler@v4
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

please add a new line here
```suggestion
        repo-token: "${{ secrets.GITHUB_TOKEN }}"
        
```